src/ensembl/src/content/app/entity-viewer/gene-view/components/collapsed-exons-image/CollapsedExonsImage.tsx

```diff
@@ -15,27 +15,22 @@
  */
```

Comment:
Regarding the naming.

1) If we change this component to represent the length of the actual protein, it should no longer be called CollapsedExonsImage.
2) I still think that it's better to use names that make sense _within a component_. Neither this component nor the ProteinDomainImage component need to be concerned about the concept of the "longest protein" (which becomes a thing only at the level of their parent component, which can compare proteins with each other). What this and the ProteinDomainImage component are interested in is the length of the whole track in amino acids. Could be called something like `trackLength`.

Reply:
I have changed the component's name to `ProteinImage`. Also, have changed `longestProteinLength` to `trackLength`.